### PR TITLE
[ Azure CDN ] Renamed cmdlets for new AFD SKUs

### PR DESCRIPTION
**Pull request by jorinmejia**
## Description

This PR renames the new AFD cmdlets from AzAfd to AzFrontDoorCdn. The changes have already been approved in the following PR : https://github.com/Azure/azure-powershell/pull/14491

This PR only changes the naming of the cmdlets, updates the naming in the tests, and regenerates the md files.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/920
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by VeryEarly — COMMENTED** [review]
rename cmdlets without give aliases are breaking changes.
These changes are only allowed in next sprint:
https://github.com/Azure/azure-powershell/milestone/122

**Comment by VeryEarly** [comment]
please update parameter: "-QueryParameters" to "-QueryParameter"

File: src/Cdn/Cdn/ChangeLog.md [hunk 1]
```diff
@@ -19,8 +19,6 @@
 -->
 
 ## Upcoming Release
-
-## Version 1.7.0
 * Added cmdlets to support new AFD Premium / Standard SKUs
```
Comment (jorinmejia) [inline src/Cdn/Cdn/ChangeLog.md:22]: @dingmeng-xue / @VeryEarly to confirm this change satisfies the ChangeLog format

Comment (jorinmejia) [inline src/Cdn/Cdn/ChangeLog.md:22]: @dingmeng-xue / @VeryEarly to confirm this change satisfies the ChangeLog format

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**